package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "yb-rrdiagram",
+  "version": "0.0.1",
+  "description": "Generate railroad diagrams and grammar for YugabyteDB YSQL/YCQL grammar",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
what is this?

Reply:
This is the default package.json generated by npm. When we add tests into the package, will update it.